config/cfme/manifest_5_11.json

```diff
@@ -84,115 +135,114 @@
-        "ram_size_in_bytes": null,
-        "retired": null,
-        "v_datastore_path": null,
-        "hardware": {
-          "id": null,
-          "disks": {
-            "id": null,
-            "device_name": null,
-            "device_type": null,
-            "disk_type": null,
-            "free_space": null,
-            "mode": null,
-            "size": null,
-            "size_on_disk": null,
-            "partitions": {
-              "id": null,
-              "name": null,
-              "controller": null,
-              "free_space": null,
-              "partition_type": null,
-              "size": null,
-              "start_address": null,
-              "used_space": null,
-              "virtual_disk_file": null,
-              "volume_group": null,
-              "volumes": {
-                "id": null,
-                "name": null,
-                "filesystem": null,
-                "free_space": null,
-                "size": null,
-                "typ": null,
-                "used_space": null,
-                "volume_group": null
-              }
-            }
-          }
-        }
+        "key": null,
+        "company": null,
+        "label": null,
+        "version": null,
       },
       "ems_clusters": {
         "id": null,
         "name": null,
-        "uid_ems": null,
+        "type": null,
         "ems_ref": null,
-        "ha_enabled": null,
-        "drs_enabled": null,
-        "effective_cpu": null,
-        "effective_memory": null
+        "v_parent_datacenter": null,
       },
       "hosts": {
         "id": null,
+        "ems_ref": null,
         "name": null,
-        "type": null,
         "hostname": null,
-        "ipaddress": null,
-        "power_state": null,
-        "guid": null,
-        "uid_ems": null,
-        "ems_ref": null,
-        "mac_address": null,
-        "maintenance": null,
-        "vmm_vendor": null,
-        "vmm_version": null,
-        "vmm_product": null,
-        "vmm_buildnumber": null,
-        "archived": null,
-        "cpu_cores_per_socket": null,
+        "type": null,
         "cpu_total_cores": null,
-        "ems_cluster": {
-          "ems_ref": null
+        "cpu_cores_per_socket": null,
+        "ems_cluster_id": null,
+        "switches": {
+          "id": null,
+          "name": null,
+          "uid_ems": null,
+          "type": null,
         },
-        "hardware": {
-          "memory_mb": null
-        }
-      },
-      "storages": {
-        "id": null,
-        "name": null,
-        "location": null,
-        "store_type": null,
-        "total_space": null,
-        "free_space": null,
-        "uncommitted": null,
-        "storage_domain_type": null,
-        "host_storages": {
+        "storages": {
+          "id": null,
+          "name": null,
           "ems_ref": null,
-          "host": {
-            "ems_ref": null
+          "store_type": null,
+          "multiplehostaccess": null,
+        },
```

Comment:
Same thing here for storages, you have it at the host level when we already collect storages at the top-level _and_ map storages to hosts.